core/market/decentralized/tests/test_resolver.rs

```diff
@@ -0,0 +1,151 @@
+#[macro_use]
+mod utils;
+
+#[cfg(test)]
+mod tests {
+    use std::{future::Future, time::Duration};
+    use tokio::time::{timeout, Timeout};
+
+    use crate::utils::{
+        client::{sample_demand, sample_offer},
+        MarketsNetwork,
+    };
+
+    /// Test adds Offer on single node. Resolver should not emit Proposal.
+    #[cfg_attr(not(feature = "market-test-suite"), ignore)]
+    #[actix_rt::test]
+    async fn test_single_not_resolve_offer() -> Result<(), anyhow::Error> {
+        // given
+        let _ = env_logger::builder().try_init();
+        let mut network = MarketsNetwork::new("test_single_resolve_offer")
```

Comment:
`test_single_resolve_offer` differs from function name `test_single_not_resolve_offer`. For purpose?

Reply:
thx